components/List/tests/snapshot/MDCSelfSizingStereoCellSnapshotTests.m

```diff
@@ -109,78 +124,112 @@ - (void)generateSnapshotWithContentSizeCategoryAndNotificationPost:
-    [self.cell setNeedsLayout];
+    cell.titleLabel.text = @"Title";
+    cell.titleLabel.font = self.typographyScheme.subtitle1;
+    cell.detailLabel.text = @"Detail";
+    cell.detailLabel.font = self.typographyScheme.button;
+    cell.mdc_adjustsFontForContentSizeCategory = YES;
+    self.arrayOfCells = @[ cell ];
 
     // Then
     [self generateSnapshotWithContentSizeCategoryAndNotificationPost:UIContentSizeCategoryExtraSmall
-                                                    andVerifyForView:self.cell];
+                                                    andVerifyForView:self.collectionView];
   }
 }
 
 - (void)testCellWithDynamicTypeForContentSizeCategoryExtraLargeEnabledForTitleAndDetail {
   if (@available(iOS 10.0, *)) {
     // Given
+    MDCSelfSizingStereoCell *cell = [[MDCSelfSizingStereoCell alloc] init];
     self.typographyScheme =
         [[MDCTypographyScheme alloc] initWithDefaults:MDCTypographySchemeDefaultsMaterial201902];
 
     // When
-    self.cell.titleLabel.text = @"Title";
-    self.cell.titleLabel.font = self.typographyScheme.subtitle1;
-    self.cell.detailLabel.text = @"Detail";
-    self.cell.detailLabel.font = self.typographyScheme.button;
-    self.cell.mdc_adjustsFontForContentSizeCategory = YES;
-    [self.cell setNeedsLayout];
+    cell.titleLabel.text = @"Title";
+    cell.titleLabel.font = self.typographyScheme.subtitle1;
+    cell.detailLabel.text = @"Detail";
+    cell.detailLabel.font = self.typographyScheme.button;
+    cell.mdc_adjustsFontForContentSizeCategory = YES;
+    self.arrayOfCells = @[ cell ];
 
     // Then
     [self generateSnapshotWithContentSizeCategoryAndNotificationPost:UIContentSizeCategoryExtraLarge
-                                                    andVerifyForView:self.cell];
+                                                    andVerifyForView:self.collectionView];
   }
 }
 
+- (NSInteger)numberOfSectionsInCollectionView:(UICollectionView *)collectionView {
+  return 1;
+}
+
+- (NSInteger)collectionView:(UICollectionView *)collectionView
+     numberOfItemsInSection:(NSInteger)section {
+  return self.arrayOfCells.count;
+}
+
+- (UICollectionViewCell *)collectionView:(UICollectionView *)collectionView
+                  cellForItemAtIndexPath:(NSIndexPath *)indexPath {
+  MDCSelfSizingStereoCell *dequeuedCell =
+      [collectionView dequeueReusableCellWithReuseIdentifier:kSelfSizingStereoCellIdentifier
+                                                forIndexPath:indexPath];
+  // Temporarily add dequeuedCell as a subview of collectionView
+  // to inherit traitCollection. The dequeuedCell will be removed
+  // from parent at the end of this method.
+  [self.collectionView addSubview:dequeuedCell];
+  MDCSelfSizingStereoCell *cell = self.arrayOfCells[indexPath.item];
+
+  // Draw border to distinguish cell from the collection view.
+  dequeuedCell.layer.borderColor = UIColor.blackColor.CGColor;
+  dequeuedCell.layer.borderWidth = 1;
```

Comment:
Rather than use a border (which might affect other aspects of the cell rendering), can we give the collection view background a distinct color?

Reply:
Addressed in 2b20b083810b2ec36477e4820fffc48341ac0235. It uses grey color as backgroundColor now.

Reply:
Ahh, can we change the collection view's background color so that the cell looks more close to what a cell would typically look like? We care less about the collection view's background color for the purpose of these tests.

Reply:
Oh. sorry. I misunderstood it. Addressed in bfe6d57a772e264c20ac342fa6c5dd81d78618ae